lib/ansible/modules/cloud/amazon/ec2_vpc_dhcp_options.py

```diff
@@ -233,7 +239,8 @@ def ensure_tags(vpc_conn, resource_id, tags, add_only, check_mode):
         latest_tags = get_resource_tags(vpc_conn, resource_id)
         return {'changed': True, 'tags': latest_tags}
     except EC2ResponseError as e:
-        module.fail_json(msg=get_error_message(e.args[2]))
+        module.fail_json(msg=to_native(e))
```

Comment:
While fixing this could just do `module.fail_json(msg="Failed to modify tags.", exception=traceback.format_exc(), e.message)` (+ import traceback) to be uniform with the boto2 guidelines.
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#boto-2